jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -812,15 +811,8 @@
       ;;   (return-from :try-to-pick-object nil))
       (setq sign 0)
       (setq sign_y -1)
-      (if (eq arm :rarm)
-        (ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a sign_y: ~a" arm sign sign_y)
-        (ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a" arm sign))
-      (if (eq arm :larm)
-        (progn
-          ;; start the vacuum gripper
-          (ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a start vacuum gripper" arm)
-          (send self :start-grasp arm)
-          (unix::sleep 1)) nil)
+      (ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a sign_y: ~a" arm sign sign_y)
+      (ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a" arm sign)
```

Comment:
`(ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a" arm sign)`
This is for old left arm.
`(ros::ros-info "[:try-to-pick-object-in-order-bin] arm:~a sign: ~a sign_y: ~a" arm sign sign_y)` is sufficient.